src/mrpro/algorithms/csm/iterative_walsh.py

```diff
@@ -45,6 +40,14 @@ def iterative_walsh(
         width of the smoothing filter
     power_iterations
         number of iterations used to determine dominant eigenvector
+
+    References
+    ----------
+    .. [1] https://github.com/ismrmrd/ismrmrd-python-tools
+    .. [2] Daval-Frerot G, Ciuciu P (2022) Iterative static field map estimation for off-resonance correction in
+    non-Cartesian susceptibility weighted imaging. MRM 88(4): mrm.29297.
```

Comment:
be careful with multiline references. the second line has to start EXACTLY under the first letter (i.e. D of Daval-Frerot) of the first line